storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -1835,6 +1835,21 @@
     public static final String TOPOLOGY_CLASSPATH="topology.classpath";
 
     /**
+     * Topology-specific classpath for the worker child process. This will be *prepended* to
+     * the usual classpath, meaning it can override the Storm classpath. This is for debugging
+     * purposes, and is disabled by default. To allow topologies to be submitted with user-first
+     * classpaths, set the user.classpath.first.enabled config to true.
+     */
+    @isStringOrStringList
+    public static final String TOPOLOGY_CLASSPATH_FIRST="topology.classpath.first";
```

Comment:
I don't think we want that one. Would `topology.classpath.prefix` or `topology.prefix.classpath` be better?

It should be `topology.*` because that's the 'namespace' that all topology configs live in.